grisera_api/recording/recording_service_graphdb.py

```diff
@@ -17,8 +17,11 @@ class RecordingServiceGraphDB(RecordingService):
     registered_channel_service(RegisteredChannelService): Service to send registered channel requests
     """
     graph_api_service = GraphApiService()
-    participation_service = ParticipationServiceGraphDB()
-    registered_channel_service = RegisteredChannelServiceGraphDB()
+
+    def __init__(self):
+        self.participation_service = None
+        self.registered_channel_service = None
+        self.observable_information_service = None
```

Comment:
Missing type definition of these services

Reply:
As above